docs/enterprise/helm-guide.md

```diff
@@ -0,0 +1,228 @@
+# Helm guide
+
+These instructions explain how to install and run our enterprise offering,
```

Comment:
`how to install and run **Feldera Enterprise** on a Kubernetes cluster. It requires a valid Feldera Enterprise license -- please request one at ....`